src/main/java/racingCar/domains/Name.java

```diff
@@ -0,0 +1,38 @@
+package racingCar.domains;
+
+import java.util.Objects;
+
+public class Name {
+    private String name;
```

Comment:
```suggestion
public class Name {
    private final String name;
```

불변 객체로 만들 수 있다면, 불변하도록 final 키워드를 붙여주시면 좋을 것 같습니다.
동작에는 차이가 없어보이지만, 개인 경험상 협업을 하며 생기는 대부분의 버그를 방지할 수 있더라고요.